circle.yml

```
@@ -539,7 +539,7 @@ jobs:
       - run:
           name: Package
           # The package must be run within the actual crate and not in the workspace.
-          working_dir: bindings/rust
+          working_dir: ~/bindings/rust
```

```suggestion
          working_dir: ~/project/bindings/rust
```
if absolute path is required.

---

Ok, it should have been `working_directory`.

---

Neither `bindings/rust`, `~/bindings/rust`, or `~/project/bindings/rust` works. Will check `pwd` because this is confusing.